lib/PuppeteerSharp/Page.cs

```diff
@@ -19,11 +19,12 @@
 namespace PuppeteerSharp
 {
     [DebuggerDisplay("Page {Url}")]
-    public class Page : IDisposable
+    public class Page : IDisposable, IEquatable<Page>
     {
         public int DefaultNavigationTimeout { get; set; } = 30000;
 
-        private readonly Session _client;
+        internal readonly Session _client;
```

Comment:
for the test ShouldNotCrashWhileRedirectingIfOriginalRequestWasMissed

Reply:
If it's internal let's make this an internal getter property.